Airports/EGFE/SMR/Geo.txt

```diff
@@ -0,0 +1,13 @@
+EGFE Haverford West			S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
```suggestion
EGFE Haverford West            S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```
I think we typically use spaces rather than tabs for this kind of thing.